middlewares/auth.js

```diff
@@ -0,0 +1,43 @@
+const passport = require("passport");
+const passportJWT = require("passport-jwt");
+const User = require("../service/schemas/user");
+require("dotenv").config();
+const secret = process.env.SECRET;
+
+const ExtractJWT = passportJWT.ExtractJwt;
+const Strategy = passportJWT.Strategy;
+const params = {
+  secretOrKey: secret,
+  jwtFromRequest: ExtractJWT.fromAuthHeaderAsBearerToken(),
+};
+
+// JWT Strategy
+passport.use(
+  new Strategy(params, function (payload, done) {
```

Comment:
```suggestion
  new Strategy(params,  (payload, done) => {
```
Może warto użyć arrow function gdyż taki sposób pisania Widzę że preferujesz

Reply:
wzięte z konspektu dlatego nie zmieniałam:)

Reply:
ale już zmienione :)